CHANGES.md

```diff
@@ -1,3 +1,9 @@
+2.0.1 (2019-06-04)
+------------------
+
+* Zed\_char: add options of combining marks for transform functions (#18)
```

Comment:
Grammar here is a bit stilted. I'd recommend new phrasing but I don't understand what the sentence means so I can't.

Reply:
Hmm, it seems my baby-level English really causes some trouble. I think I'll spend some time to improve it to, at least, child-level.